server/app/controllers/monitoring/MetricsController.java

```diff
@@ -102,35 +104,4 @@ public Result getMetrics() {
 
     return ok(writer.toString()).as(TextFormat.CONTENT_TYPE_004);
   }
-
-  @VisibleForTesting
-  static void initializeCounters() {
-    QUERY_METRIC_COUNT =
-        Counter.build()
-            .name("ebean_queries_total")
-            .help("Count of database queries")
-            .labelNames("name", "location", "className")
-            .register();
-
-    QUERY_METRIC_MEAN_LATENCY =
-        Counter.build()
-            .name("ebean_queries_mean_latency_micros")
-            .help("Mean latency of database queries in micros")
-            .labelNames("name", "location", "className")
-            .register();
-
-    QUERY_METRIC_MAX_LATENCY =
-        Counter.build()
-            .name("ebean_queries_max_latency_micros")
-            .help("Max latency of database queries in micros")
-            .labelNames("name", "location", "className")
-            .register();
-
-    QUERY_METRIC_TOTAL_LATENCY =
-        Counter.build()
-            .name("ebean_queries_total_latency_micros")
-            .help("Total latency of database queries in micros")
-            .labelNames("name", "location", "className")
-            .register();
```

Comment:
Moved these to a separate class and is configured as a singleton via DI. Having them static continued to be an issue with the tests.

Reply:
Centralizing it would probably be good. But I'd say that would be a different PR.